go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -222,11 +233,19 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb, sourceIP string, sourcePo
 		klog.Errorf("Error while querying ACLs by name: %s, %v", stderr, err)
 	} else if len(aclUUID) > 0 {
 		klog.Infof("Existing Service Reject ACL found: %s for %s", aclUUID, aclName)
-		_, stderr, err = util.RunOVNNbctl("--", "add", "port_group", ovn.clusterPortGroupUUID, "acls", aclUUID)
-		if err != nil {
-			klog.Errorf("Failed to add LB %s ACL %s %q to cluster port group: %q (%v)",
-				lb, aclUUID, aclName, stderr, err)
-			return "", err
+		var cmd []string
+		if applyToPortGroup {
+			cmd = append(cmd, "--", "add", "port_group", ovn.clusterPortGroupUUID, "acls", aclUUID)
+		}
+		if len(gwRouterExtSwitch) > 0 {
+			cmd = append(cmd, "--", "add", "logical_switch", gwRouterExtSwitch, "acls", aclUUID)
```

Comment:
it might work, but it will end up installing lot of flows on the external switch as well for normal service reject acl and that the last thing we want. Assuming that reject acl for NodePort/External Ip will be installed on specific external switch of specific node, this makes sense to me.

Reply:
yeah management port doesn't go on the external switch, only node switch